project1/register.py

```diff
@@ -0,0 +1,28 @@
+
+import time
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+
+
+db = SQLAlchemy()
+
+class User(db.Model):
+	__tablename__ = "users"
+	Username = db.Column(db.String, unique=True,primary_key=True)
+	Email = db.Column(db.String, unique=True, nullable=False)
+	Gender = db.Column(db.String,nullable=False)
+	Password = db.Column(db.String, nullable=False)
+	Cpassword = db.Column(db.String, nullable=False)
+	Time_registered = db.Column(db.DateTime, nullable=False)
+
+	def __init__(self,Username,Email,Gender,Password,Cpassword,Time_registered):
+		self.Username=Username
+		self.Email = Email
+		self.Gender=Gender
+		self.Password = Password
+		self.Cpassword = Cpassword
```

Comment:
what's the use of Cpassword?

Reply:
Cpassword is for confirmation of password, i thought it maight be one of the field of registartion page.

Reply:
This creates redundant data in database, it'd be better to verify that as a part of registration function and not update it into the database.

Reply:
ok, but not s the database is created, it would be difficult to remove this. Although it can be useful to that user can type correct password, incase he is mistyped. i will implement it such that the user can type the password correctly.